tests/utils.py

```diff
@@ -817,3 +817,20 @@ def load_nist_kbkdf_vectors(vector_data):
             test_data[name.lower()] = value.encode("ascii")
 
     return vectors
+
+
+def load_ed25519_vectors(vector_data):
+    data = []
+    for line in vector_data:
+        x = line.split(':')
+        secret_key = x[0][0:64]
+        public_key = x[1]
+        message = x[2]
+        signature = x[3][0:128]
```

Comment:
This could all be written better with some unpacking `secret_key, public_key, message, signature = line.split(":")`